scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/client/SchedulerNodeClient.java

```diff
@@ -299,6 +299,19 @@ public JobId submit(File job)
         return client.submit(job);
     }
 
+    @Override
+    public JobId submit(File job, Map<String, String> variables)
+            throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
+        return submit(job, variables, null);
+    }
+
+    @Override
+    public JobId submit(File job, Map<String, String> variables, Map<String, String> genericInfos)
+            throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
+        renewSession();
+        return client.submit(job, variables, genericInfos);
+    }
+
     @Override
     public JobId submit(URL job)
             throws NotConnectedException, PermissionException, SubmissionClosedException, JobCreationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)